embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/comms/delivery/v2/CacheService2.kt

```diff
@@ -0,0 +1,8 @@
+package io.embrace.android.embracesdk.internal.comms.delivery.v2
+
+import java.io.File
+
+interface CacheService2 {
+    fun deletePayload(file: File)
+    fun getPayloads(priority: Int): List<File>
```

Comment:
What are the possible values of `priority`? Could an enum better represent these states?

Reply:
oh definitely enum. Again, I was just trying to clobbering something up for testing purposes. I should've specified as much